Apps/BabylonScripts/Build/DownloadAndCopyFiles.ps1

```diff
@@ -0,0 +1,24 @@
+Push-Location $PSScriptRoot
+Write-Host "Installing npm packages..."
+npm install
+
+$files = "babylon.gui.js",

```

Comment:
I wonder if you could do some `Get-ChildItem` fanciness and glob *.js, *.js.map, *.d.ts instead of needing to explicitly list the files.

Reply:
I don't think we'll ever get away from needing to list specific file requirements. I think this is okay for the time being.